site/src/pages/GroupsPage/GroupPage.tsx

```diff
@@ -173,7 +173,6 @@ export const GroupPage: React.FC = () => {
                               <AvatarData
```

Comment:
It looks like before the avatars on the group page would overlap a little but now they touch but do not look like they are overlapping.  Was that intentional?  I think the overlap looks kinda nice.

Reply:
Good catch! I'm fixing this.